docs/code_ref/sunpy_stability.yaml

```diff
@@ -26,17 +26,20 @@ physics:
   status: dev
   comments:
 roi:
-  status: planned
-  comments: There is very little functionality.
+  status: deprecated
+  comments: Will be removed in sunpy 3.1. There is little functionality.
 sun:
-  status: dev
-  comments: Sub-module `constants` is stable.  Sub-module `models` will likely have major changes.
+  status: stable
+  comments: Any future changes should largely maintain backwards compatibility.
```

Comment:
I think models still needs an overhaul, so I would keep this as is?

Reply:
Fair. I will make the change.